src/models/item.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const itemSchema = new Schema({
+  name: {
+    type: String,
+    maxLength: [30, "item name's max length is 30 characters"],
```

Comment:
A better error message could be: `Item name should have max length of 30 characters only.`